BinaryEncoding.md

```diff
@@ -81,7 +81,7 @@ Yes:
   - A table (sorted by offset) containing, for each type which has opcodes:
     + A standardized string literal [type name](AstSemantics.md#local-and-memory-types).
       The index of a type name in this table is referred to as a type ID
-    + 32-bit offset of its opcode table within the section
+    + 64-bit offset of its opcode table within the section
```

Comment:
I'm absolutely certain we don't need 4gb here. I'm not going to argue over what will be a small handful of bytes, but I just don't see the necessity